pineappl_cli/tests/convolve.rs

```diff
@@ -15,8 +15,6 @@ Options:
   -b, --bins <BINS>       Selects a subset of bins
   -i, --integrated        Show integrated numbers (without bin widths) instead of differential ones
   -o, --orders <ORDERS>   Select orders manually
-      --xir <XIR>         Set the variation of the renormalization scale [default: 1.0]
-      --xif <XIF>         Set the variation of the factorization scale [default: 1.0]
```

Comment:
is this a temporary change? because this feature is actually quite useful

Reply:
Maybe I'm mistaken, but it seems @Radonirinaunimi removed the feature during the merge. You should also provide the possibility to vary to fragmentation scale.